docs/UserGuide.md

```diff
@@ -291,9 +287,11 @@ Format: `reminder NUMBER TIME_UNIT`
 
 **:information_source: Notes about the reminder command:**<br>
 
-* `TIME_UNIT` must be either `days` or `weeks`. `day` and `week` **will not work**.
+* `NUMBER` **can be any integer** …​, -7, -3, 0, 3, 7, …​
+* `TIME_UNIT` must be either `day(s)` or `week(s)`.
+  * `NUMBER` in the range of **[-1, 0, 1]** must have`TIME_UNIT` to be **singular**
```

Comment:
This one no need already. Can change to 
`TIME_UNIT` must be either `days` or `weeks`.
`day` or `week` accepted when number is in the range of **[-1, 0, 1]**

Reply:
All right noted.